Automattic/jetpack
# Build: Base webpack.config.js on calypso-build's

#### Changes proposed in this Pull Request:

Build: Base webpack.config.js on calypso-build's

This allows us to remove a few now-redundant Gulp tasks, since `calypso-build` takes care of e.g. minification, building CSS from SCSS, and RTL for us automatically.

#### Testing instructions:

```
yarn distclean && yarn
yarn build-client
```

Builds without errors?

```
yarn docker:up
```

Smoke-test the React Dashboard. Watch the browser console for errors.

Also try

```
yarn clean-client && NODE_ENV=production yarn build-client
```

and verify that `_inc/build/admin.js` is minified.

#### Proposed changelog entry for your changes:

* Build: Base webpack.config.js on calypso-build's
